package.json

```diff
@@ -4,10 +4,11 @@
   "description": "Django website for canonical.com",
   "main": "index.js",
   "scripts": {
-    "build": "node-sass --include-path node_modules --include-path static/sass static/sass --output static/css && postcss --use autoprefixer -b 'last 2 versions' < static/css/styles.css | postcss --use cssnano > static/css/styles.min.css",
-    "watch": "node-sass --include-path node_modules --source-map true --watch static/sass --output static/css",
+    "build": "yarn run copy-3rd-party-js && node-sass --include-path node_modules static/sass --output static/css && postcss --use autoprefixer --replace 'static/css/**/*.css' && postcss --use cssnano --dir static/minified 'static/css/**/*.css'",
+    "copy-3rd-party-js": "cp node_modules/cookie-policy/build/js/cookie-policy.js static/js/modules/ && cp node_modules/global-nav/build/js/global.js static/js/modules/",
```

Comment:
I thought you were going to copy the 3rd party scss into the static directory too?